ansible/roles/davros/files/supervisor/davros.conf

```diff
@@ -2,6 +2,7 @@
 directory = /opt/davros
 command = docker-compose up --no-color
 redirect_stderr = true
+autorestart = true
```

Comment:
Why do we need this? Does it die unexpectedly? Maybe that'a bug we should fix.

Reply:
Yeah, I remember it was something like that. I'll remove the line and start another issue if it persists.